README.md

```diff
@@ -279,14 +588,15 @@ Please see [our examples for a complete list][examples] of tested targets and dr
 ## How can I contribute?
 
 The easiest way for you and the best way for us to see if something is unclear
-or missing, is if you use the library and give us some feedback by
-[filing a bug report][issues] or if you have a fix already, [opening a pull request][prs].
+or missing, is if you use the library and give us some feedback by [filing a bug
+report][issues] or if you have a fix already [opening a pull request][prs].
 
 See [CONTRIBUTING.md][contrib] for our contribution guidelines.
 
 <!--authors-->
 The modm project is maintained by
-Niklas Hauser ([\@salkinium](https://github.com/salkinium)) with significant contributions from
+Niklas Hauser ([\@salkinium](https://github.com/salkinium)) and
+Raphael Lehmann ([\@rleh](https://github.com/rleh)) with significant contributions from
 Sascha Schade ([\@strongly-typed](https://github.com/strongly-typed)),
```

Comment:
This list could be updated to reflect https://github.com/modm-io/modm/graphs/contributors.

Reply:
Yes, done in #597.

Reply:
Also GitHub seems to ignore the `.mailmap` file sadly, so the graph isn't entirely accurate. For example, Sascha actually has ~850 commits, not ~300 as indicated, because he used `<strongly-typed@nan>` as an email for some time (which is not even a valid email URL 🤦‍♂️).

Reply:
Also, the list in #597 does not fully match https://github.com/modm-io/modm/graphs/contributors, ` dhebbeker` is missing for example. In my opinion a reasonable approach would be making this identical to the github contributor list down to a certain commit count.

Reply:
Yes, that was my intention, but I choose 100 as an arbitrary point… What's significant for us? >20?

```
# on the feature/2021q1 branch with updated .mailmap
git shortlog -sn
  2778  Niklas Hauser
   851  Sascha Schade
   597  Kevin Läufer
   572  Fabian Greif
   161  Martin Rosekeit
   161  Daniel Krebs
   159  Georgi Grinshpun
   120  Raphael Lehmann
   100  Christopher Durand
    66  David Hebbeker
    39  Thorsten Lajewski
    26  Mike Wolfram
    17  Antal Szabó
    15  Erik Henriksson
    11  Michael Thies
    11  Thomas Sommer
    10  Carl Treudler
     9  Pavel Pletenev
     8  Sebastian Birke
     8  Benjamin Carrick
     7  Jeff McBride
     4  Vivien Henry
     3  Álan Crístoffer
     3  Benjamin Weps
     3  Christian Menard
     3  Kaelin Laundry
     3  Marten Junga
     3  Nick Sarten
     3  Niclas Rohrer
     2  Lucas Mösch
     2  Linas Nikiperavicius
     2  Amarok McLion
     2  Nicolai Bruhn
     2  Jacob Schultz Andersen
     2  Felix Petriconi
     2  Tarik TIRE
     1  Julia Gutheil
     1  Patrick Servello
     1  Lukas Güldenstein
     1  Philipp Graf
     1  Hans Schily
     1  Andre Gilerson
     1  Sascha Schade (strongly-typed)
     1  Christoph Rüdi
     1  Sergiy Yevtushenko
     1  Martin Esser
     1  Thomas Figueroa
     1  Cerem Cem ASLAN
     1  Jörg Hoffmann
     1  Zawadniak Pedro
     1  Arjun Sarin
     1  Tomasz Chyrowicz
     1  Niklas Meyer
```

Reply:
Oh, this list is entirely different to the github generated one ... Is it because of the old xpcc commits with invalid email addresses? It seems the count is off for RCA members who contributed to xpcc.

Reply:
It's because GitHub only assigns the commits based on the email address associated with the account. However, several people used different email addresses (also to distinguish between contributions in their free time and their job), which is tracked in `.mailmap` and used by Git to determine the actual person.

Reply:
Let's continue this discussion in #597.

Reply:
Ah, ok.

> What's significant for us? >20?

Don't really know... What ever metric we choose, it will never really resemble the actual effort and impact of changes done. Adding fiber support to modm is one commit, adding a Nucleo board blink example is as well...